staging/publishing/rules.yaml

```diff
@@ -1598,3 +1598,17 @@ rules:
       branch: release-1.18
     - repository: metrics
       branch: release-1.18
+
+- destination: controller-manager
+  library: true
+  branches:
+  - source:
+      branch: master
+      dir: staging/src/k8s.io/controller-manager
+    name: master
+  - source:
+      branch: release-1.19
```

Comment:
Don't think this branch exists yet, that might be why verify is failing

Reply:
This is failing because the publishing-bot config references the https://github.com/kubernetes/controller-manager repo but the repo doesn't exist yet.

@cheftako can you create an issue in k/org to create the repo? I can get the repo setup. Once that's done, verify should be green and publishing should be good to go.

---

> Don't think this branch exists yet, that might be why verify is failing

Even if the branch doesn't exist in a repo, the publishing-bot creates branches mentioned in the config. :)

Reply:
Fixed